book/doctrine.rst

```diff
@@ -145,6 +141,54 @@ for you:
         collation-server     = utf8mb4_general_ci # Replaces utf8_general_ci
         character-set-server = utf8mb4            # Replaces utf8
```

Comment:
Shouldn't we keep this so that the MySQL settings are the same accessing the database using other clients?